apps/accounts/views.py

```diff
@@ -161,7 +161,7 @@ def get(self, request, user, identity, next_path):
             return HttpResponseRedirect(path)
         else:
             login_user(request, user, identity)
-            next_path = next_path or '/'
+            next_path = next_path or reverse('home')
             log.info('Redirecting after login to: {}'.format(next_path))
             return HttpResponseRedirect(next_path)
```

Comment:
I added a `safe_redirect()` function here that will check again but I left the original check in `@with_user` because sanitizing the URL when it is decoded seems like the right thing to do to me.